### DISCUSSION TITLE
why does termux install gcc 11 and g++ 11 instead of gcc 9 and g++ 9

### DISCUSSION THREAD

**Discussion by garzeth** [termux/termux-app#2792] [Q&A]
I have Ubuntu 20.04 on termix and If I know the default packages and compilers rae gcc 9 and g++9 and openssl 1.1.1f but why it suddenly install latest gcc

**Comment by ChiefMikeK** [comment]
Due to cloud/repo hosting storage limits only one package version is available

**Reply by garzeth** [reply]
thanks for answer but I think I seem to fix the problem by using update-alternatives
